### PR TITLE
Feature/68 implementing footer

### PR DESCRIPTION
**Pull request by Eheijboer**
## Description

implementing the footer on all pages.

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I did not update API Controllers, if I did, I added/changed Postman tests
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] New and existing unit tests pass locally with my changes
-   [x] I updated the changelog with an end-user readable description
-   [x] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

After running npm install footer should be visible at the bottom of each page.

## Link to issue

Closes: #68

### CODE REVIEW

**Comment by Brend-Smits** [comment]
Next time please fill in where people who test this pull request can find this change. It might be obvious for you, but not to the person testing the pull request. This is why the section "Steps to Test or Reproduce" is in the template. 
In this section you also mention if there are any edge cases (perhaps in this case if there are places where the footer should not be shown).

The description was also not added describing what the pull request is here for and what it adds.

File: src/app/components/footer/footer.component.scss [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+.footer-distributed .footer-links a:before {
+  content: "·";
+  font-size: 20px;
+  left: 0;
+  color: #fff;
+  display: inline-block;
+  padding-right: 5px;
+}
```
Comment (MaxKorlaar) [inline src/app/components/footer/footer.component.scss]: These styles do not seem to be used anywhere.
```suggestion

```

**Comment by MaxKorlaar** [comment]
Looks good, however it is not fully responsive yet. On mobile devices, it looks a bit off. I don't know if that's a priority for now and/or this issue @Brend-Smits.
